hi-ml/src/health_ml/utils/checkpoint_handler.py

```diff
@@ -126,6 +126,10 @@ def get_local_checkpoints_path_or_download(self) -> Path:
             download_folder.mkdir(exist_ok=True, parents=True)
             checkpoint_path = CheckpointHandler.download_weights(url=self.container.weights_url,
                                                                  download_folder=download_folder)
+        elif self.container.ckpt_run_id:
+            downloader = get_checkpoint_downloader(self.container.ckpt_run_id, self.container.outputs_folder)
+            checkpoint_path = downloader.local_checkpoint_path
+            assert checkpoint_path.is_file(), f"Couln't download checkpoint from run {self.container.ckpt_run_id}."
         else:
             raise ValueError("Cannot download weights, neither local_weights_path or weights_url are set")
```

Comment:
This error message needs to inlcude ckpt_run_id

Reply:
Sure